<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_imagen_original">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>imagen original</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imagen_procesada">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>imagen procesada</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PROCESAMIENTO DE IMAGENES</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Espacio de color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_cargar_imagen">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar imagen</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_espacio_color">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_aplicar_color">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aplicar cambio de espacio de color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_ecualizar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ecualizar histograma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_binarizar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Binarizar Imagen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamaño de kernel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_morfologia">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aplicar morfología</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_contar">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Contar células</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_filtro_extra">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Metodo extra (filtro bilateral)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_reiniciar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restaurar imagen original</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_volver">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver al menu principal</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_umbral">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Umbral:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_kernel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Operación morfológica:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_morfologia">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_resultado_celulas">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
